app/api/__init__.py

```diff
@@ -72,6 +76,13 @@
 api.route(TrackRelationship, 'track_sessions', '/tracks/<int:id>/relationships/sessions')
 api.route(TrackRelationship, 'track_event', '/tracks/<int:id>/relationships/event')
 
+# event_copyright
+api.route(EventCopyrightList, 'event_copyright_list', '/events/<int:id>/event_copyright')
```

Comment:
@SaptakS  Event_copyright is one to one right, list is not needed afaik and using <int:id> instead of <int:event_id> would give list according to the id of event_copyright and not the event, the `'/events/<int:event_id>/event_copyright'` in the line below does the same work instead